src/Propulsion.SqlStreamStore/Propulsion.SqlStreamStore.fsproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
if you prefer, we can remove the `net461` and only add in the unlikely event someone happens to need it? (and maybe consider Microsoft.Data.SqlClient?)

Reply:
1. Your choice, I don't really see the need to target net461 anymore myself, just kept with what other projects were doing.
2. I'll drop SqlClient dependency from there, it's not a business of this project to restrict it, it's only there for Dapper which I believe supports both versions.

Reply:
Its definitely not a prerequisite, and if it involves conditionals and/or extra helpers, that's definitely a reason to consider removing it.
The blanket support for net461 made a lot more sense 2 years ago and for libraries that would be referenced from an app's "core" modules - realistically anything using `Propulsion.SqlStreamStore` should be a small new separated console app.

Reply:
Ok, if it's not necessary, I'll drop support for it.